src/routes/Liquidity/Liquidity.js

```diff
@@ -2,21 +2,49 @@ import React from 'react';
 
 import {ChoiceItemType, MultiChoiceMenu} from '../../components/UI';
 import {useLiquidityProviders, useLiquidityTranslation} from '../../hooks';
-import {openInNewTab} from '../../utils';
+import {useL2Wallet} from '../../providers/WalletsProvider';
+import {openInNewTab, evaluate} from '../../utils';
 
 export const Liquidity = () => {
+  const {account: accountL2} = useL2Wallet();
   const {titleTxt, descriptionTxt} = useLiquidityTranslation();
   const liquidityProviders = useLiquidityProviders();
-  const mapLiquidityProviders = () => {
-    return liquidityProviders.map(p => {
-      return {
-        ...p,
-        type: ChoiceItemType.LINK,
-        onClick: () => {
-          openInNewTab(p.url);
-        }
-      };
+
+  const dynamicQsValues = {
+    accountL2
+  };
+
+  const buildDynamicURL = (url, qsParams, dynamicQsValues = {}) => {
```

Comment:
Why did you moved `buildDynamicURL` from utils?

Reply:
fixed